src/signer/op-signer/launcher.star

```diff
@@ -0,0 +1,272 @@
+_ethereum_package_constants = import_module(
+    "github.com/ethpandaops/ethereum-package/src/package_io/constants.star"
+)
+
+_file = import_module("/src/util/file.star")
```

Comment:
This is unused. Should we remove it or leave it around so it can be used later?

Reply:
Good catch, will remove!